bindings/python/requirements.txt

```diff
@@ -1,3 +1,3 @@
-# Apache License 2.0 
+# Apache License 2.0
 
-open-space-toolkit-physics~=0.5,>=0.5.27
+open-space-toolkit-physics~=0.8.0
```

Comment:
This is locking the deps to `open-space-toolkit-physics~=0.8,<0.9`. Which makes sense until we are in the `0.x.y` version realm.

Maybe we should actually move to a `1.x.y` range?

Reply:
> Which makes sense until we are in the 0.x.y version realm

Yeah that was the goal for now

> Maybe we should actually move to a 1.x.y range?

Yeah I think we should probably transition to see soon